hrjobcontract/tests/phpunit/fabricators/HRJobContractFabricator.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+  class HRJobContractFabricator {
+
+    private static $default = [
+      'sequential' => 1
+    ];
+
+    private static $defaultDetails = [
+      'period_start_date' => '2016-01-01'
+    ];
+
+    public static function fabricate($params, $detailsParams) {
+      if (!isset($params['contact_id'])) {
+        throw new Exception('Specify contact_id value');
+      }
+
+      $contract = civicrm_api3('HRJobContract', 'create', array_merge(self::$default, $params))['values'][0];
+
+      if (isset($detailsParams)) {
```

Comment:
I'm curious why Details are included into this Fabricator instead of having its own? It looks inconsistent comparing to other entities.